File: src/test/resources/js/tests/node/germanLabel.js
function germanLabel($value) {
    let results = [];
    let p = TermFactory.namedNode("http://example.org/ns#germanLabel");
    let s = $data.find($value, p, null);
    for (let t = s.next(); t; t = s.next()) {
        let object = t.getObject();
        if (object.getTermType() !== "Literal" || !object.getLanguage().startsWith("de")) {
            results.push({
                value: object
            });
        }
    }
    return results;
}

File: src/test/resources/py/tests/node/german_label.py
def german_label(value):
    results = []
    p = py_tf.namedNode("http://example.org/ns#germanLabel")
    s = _data.find(value, p, None)
    while t := s.next():
        object = t.getObject()
        if (object.getTermType() != "Literal") | (not object.getLanguage().startswith("de")):
            results.append({"value": object})
    return results
